src/main/java/uk/gov/pay/products/service/PaymentUpdater.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.pay.products.service;
+
+import com.google.inject.persist.Transactional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.pay.products.persistence.dao.PaymentDao;
+
+import javax.inject.Inject;
+
+import static java.lang.String.format;
+
+public class PaymentUpdater {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PaymentUpdater.class);
+    
+    public static final String REDACTED_REFERENCE_NUMBER = "****";
```

Comment:
Are we sure about using 4 asterisks rather than 16? We’ve used 16 previously and the email we send to services state we use 16 so I’m too keen on changing this without a good reason.